packages/app-utils/tsconfig.json

```diff
@@ -4,7 +4,13 @@
     "rootDir": "src/",
     "outDir": "dist/"
   },
-  "include": ["src/**/*.ts", "src/**/*.tsx", "src/**/*.js", "src/**/*.jsx"],
+  "include": [
+    "src/**/*.ts",
+    "src/**/*.tsx",
+    "src/**/*.js",
+    "src/**/*.jsx",
+    "../plugin/src/usePlugins.ts"
```

Comment:
Is this include intentional?

Reply:
Yea seems like this was unintentional

Reply:
Ya VSCode's move file added this when I said to update import paths